ansible/roles/qe_sap_storage/tasks/generic_tasks/prepare_storage.yml

```diff
@@ -3,12 +3,42 @@
 - debug:
     msg: "{{ item }} "
 
+- name: Determine the root device
+  command: "findmnt -n -o SOURCE /"
+  register: root_device
+  changed_when: false
+
+# Cut any partition suffixes like p1, p2
+- name: Normalize root device name
+  set_fact:
+    root_device_name: "{{ root_device.stdout | regex_replace('p[0-9]+$', '') }}"
+
+# If root is not nvme0n1, and it also exists in pv, switch it with nvme0n1 in pv
+- name: Adjust PV list for storage profile "{{ item.key }}"
+  set_fact:
+    adjusted_pv: "{{ item.value.pv | map('regex_replace', '^' ~ root_device_name ~ '$', '/dev/nvme0n1') | list }}"
+  when: 
+    - root_device_name != '/dev/nvme0n1'
+    - root_device_name in item.value.pv
+
+- debug:
```

Comment:
Please add `name:` even if it is only a debug

Reply:
I copied the first debug, that was already there:

- debug:
    msg: "{{ item }} "
    
 should i change that too?